src/QMCHamiltonians/tests/CMakeLists.txt

```diff
@@ -23,9 +23,20 @@ EXECUTE_PROCESS(COMMAND ${CMAKE_COMMAND} -E make_directory "${UTEST_DIR}")
 MAYBE_SYMLINK(${UTEST_HDF_INPUT} ${UTEST_DIR}/pwscf.pwscf.h5)
 
 
-SET(SRCS test_bare_kinetic.cpp test_coulomb_pbcAB.cpp test_coulomb_pbcAB_ewald.cpp test_coulomb_pbcAA.cpp
-         test_coulomb_pbcAA_ewald.cpp test_force.cpp test_force_ewald.cpp test_ecp.cpp
-         test_hamiltonian_pool.cpp test_hamiltonian_factory.cpp test_PairCorrEstimator.cpp)
+SET(SRCS test_bare_kinetic.cpp
+	 test_coulomb_pbcAB.cpp
```

Comment:
There are tabs in this file.

Reply:
Oops. Forgot to run clang-format on this file. Fixed.

Reply:
Note to self: do not run clang-format on CMakeLists.txt